### ISSUE TITLE
How to NOT generate javadoc on annotations (eg Lombok)

### ISSUE THREAD

**Issue by mwensveen-ebpi** [mkesting/jautodoc#11]
Hi,
I use Lombok to generate getters/setters. But the Jautodoc generates javadoc for the annotatons.
Is it possible to have Jautodoc ignore annotations?

[MyClass.java_afterJautodoc.txt](https://github.com/mkesting/jautodoc/files/10896465/MyClass.java_afterJautodoc.txt)
[MyClass.java_beforeJautodoc.txt](https://github.com/mkesting/jautodoc/files/10896466/MyClass.java_beforeJautodoc.txt)

**Comment by mkesting** [comment]
I can not reproduce this. JAutodoc has an option "Comment Methods" but "Exclude [G,S]etter". Does it help when you check this option?

**Comment by mwensveen-nl** [comment]
Yes selecting the "Exclude [G,S]etter" option works for the javadoc that is generated in my example.
But when adding the annotion @AllArgsConstructor gives simular problems

```
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter

/**
 * Instantiates a new MyClass.
 *
 * @param prop1 - String
 * @param prop2 - String
 */
@AllArgsConstructor
public class MyClass {

    private String prop1;
    private String prop2;

    public MyClass() {
    }

}

```

**Comment by hazendaz** [comment]
Have same issue for years.  If any lombok usage which is a java agent (code doesn't physically exist in source), jautodoc gets incredibly confused.  I think to solve, when applying javadocs, it needs to look to see if its being placed over @Getter, @Setter, @Data, or generally any lombok.* annotation.  Probably a post check and if that is the case, don't apply it.  Now sure how that might look to fix but possible idea.

**Comment by mkesting** [comment]
Fixed with v1.16.0
